src/hello-world/index.js

```diff
@@ -5,18 +5,43 @@ import apiFetch from '@wordpress/api-fetch';
 import { useState, useEffect } from '@wordpress/element';
 import { Button, Panel, PanelBody, PanelRow } from '@wordpress/components';
 
+/**
+ * External dependencies
+ */
+import wpcom from 'wpcom';
+import wpcomOAuth from 'wpcom-oauth-cors';
+
 const rootURL = 'https://ma.tt/wp-json/';
 apiFetch.use( apiFetch.createRootURLMiddleware( rootURL ) );
 
 const HelloWorld = () => {
 	const [ posts, setPosts ] = useState( [] );
+	const [ wpcomData, setWpcomData ] = useState( [] );
 
 	useEffect( () => {
 		apiFetch( { path: '/wp/v2/posts' } ).then( ( response ) => {
 			setPosts( response );
 		} );
 	}, [] );
 
+	useEffect( () => {
+		wpcomOAuth( '67055' );
```

Comment:
@tyxla I'm wondering if it's safe to keep the `client_id` public? I'm not sure if it's the same property, but [in Calypso it seems to be available publicly](https://github.com/Automattic/wp-calypso/blob/10effec63507b9284037725d3f4e5ba5eac6dfbc/config/production.json#L147).

Reply:
Yeah, that should be fine for now. We can just keep it in a config file, but that's just fine for a proof of concept as-is.